build/integration.sh

```diff
@@ -11,7 +11,10 @@ set -o pipefail
 # Management server type. Valid values are "ads", "xds", "rest"
 XDS=${XDS:-ads}
 
-(bin/test --xds=${XDS} -debug "$@")&
+# Number of RTDS layers.
+[ "$XDS" = "ads" ] && RUNTIMES=2 || RUNTIMES=1
```

Comment:
nit: I feel this is a bit hard to read, could you use if else here?

Reply:
for sure!

Reply:
You can run with two runtimes in xDS mode I think. The missing request for the second runtime won't stop the server from responding.